core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -316,21 +318,25 @@ class ReassignPartitionsCommand(zkUtils: ZkUtils, proposedAssignment: Map[TopicA
   }
 
   def maybeLimit(throttle: Long) {
+    // Limit the throttle on currently moving replicas. Note that this command can use used to alter the throttle,
+    // but it may not alter all limits originally set, if some of the brokers have completed their rebalance.
     if (throttle >= 0) {
       val existingBrokers = existingAssignment().values.flatten.toSeq
       val proposedBrokers = proposedAssignment.values.flatten.toSeq
       val brokers = (existingBrokers ++ proposedBrokers).distinct
 
       for (id <- brokers) {
         val configs = AdminUtils.fetchEntityConfig(zkUtils, ConfigType.Broker, id.toString)
-        configs.put(DynamicConfig.Broker.ThrottledReplicationRateLimitProp, throttle.toString)
+        configs.put(DynamicConfig.Broker.ThrottledLeaderReplicationRateProp, throttle.toString)
+        configs.put(DynamicConfig.Broker.ThrottledFollowerReplicationRateProp, throttle.toString)
```

Comment:
I think it makes sense to keep the command-line tool simple while having the flexibility built-in. Because convenience is important for command-line tools, even if we do add the ability to set the throttles separately, it makes sense to keep the single switch as well. So, we can evolve this, if necessary, later.